src/status_im2/contexts/wallet/common/temp.cljs

```diff
@@ -113,6 +115,13 @@
    {:name :optimism :short-name "opt"}
    {:name :arbitrum :short-name "arb1"}])
 
+(def networks-list
```

Comment:
I think we can remove this and instead use the sub `:wallet/network-details` it has all of this info in this structure anyway. We are also not using zksync and polygon yet - as far as I know. 
This is right @ulisesmac ? 🤔 I believe you checked this recently with design team

Reply:
As a result, I removed this `networks-list` and used `:wallet/network-details` sub